FSharp/FSharp.csproj

```diff
@@ -18,7 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.0.0" />
```

Comment:
It seemed to me that there was no longer a need for this dependence. It wasn't useful here :) https://github.com/ashmind/mirrorsharp/pull/190/files#diff-0ffdce1680deddb3503495b1b9b96a4ae0a61e3813fcd6e865bcc1b5a17f5bb2L19